Sources/Application/View/View.swift

```diff
@@ -18,28 +18,52 @@ public class View: Viewable {
 		return Int(maxNumberOfGuesses)!
 	}
 
+	public func promptWordSelectionType() {
+		io.display("Would you like a randomly selected word or phrase, or would you like a friend to select one?")
+		io.display("[1] Random")
+		io.display("[2] Select")
+	}
+
+	public func receiveSelectionType(numberOfOptions: Int) -> Int{
```

Comment:
I mentioned that the reason this is more ambiguous is so that I can re-use the logic if I have another place where user selection would be required. At this time, there is only the one case, but this is extendable if necessary.